apps/website/src/lib/api/db/tables.ts

```diff
@@ -13,6 +13,11 @@ export type Course = {
   shortDescription: string,
   title: string,
   units: string[],
+  cadence: string,
+  level: string,
+  averageRating: number | null,
+  publicLastUpdated: string | null,
+  // numGraduates: number, // TODO
```

Comment:
`numGraduates` is not used yet (neither is `publicLastUpdated`), but I wanted to add all the fields that we ultimately want to have on the table. `numGraduates` is blocked by me getting access to the Applications base, see [comment here](https://github.com/bluedotimpact/bluedot/issues/714#issuecomment-2830091760)